src/common/components/animations/AnimatedTextBubble.tsx

```diff
@@ -13,7 +13,7 @@ export type Props = {
   fill?: string;
 };
 
-const AnimatedTextBubble = (props: Props) => {
```

Comment:
`React.FunctionComponent<Props>` adds `children` as an optional property. If the component doesnt support `children` prop, it adds an unnecessary property to the component.

I personally prefer setting `children` as an explicit property when declaring the property types. Doing so, the components that support children are always explicit about it.

```javascript
// example of prop types for component with children
type Props = {
  fill?: string;
  children: React.ReactNode;
}
```